Classes/Controller/Cart/CartController.php

```diff
@@ -125,7 +126,17 @@ public function showAction(
             $currentStep = (int)$this->request->getArgument('step');
         }
 
-        if ($currentStep && $this->request->getMethod() === 'POST') {
+        $currentStepHasError = false;
+        if ($extbaseAttribute instanceof ExtbaseRequestParameters
+            && $extbaseAttribute->getOriginalRequestMappingResults()
+            && $extbaseAttribute->getOriginalRequestMappingResults()->hasErrors()) {
```

Comment:
Little code style improvement to easier catch that the condition block is closed:
```suggestion
            && $extbaseAttribute->getOriginalRequestMappingResults()->hasErrors()
        ) {
```

Reply:
Done.